x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_fetch_alerts.tsx

```diff
@@ -181,9 +181,15 @@ const useFetchAlerts = ({
               next: (response: RuleRegistrySearchResponse) => {
                 if (isCompleteResponse(response)) {
                   const { rawResponse } = response;
+                  let stringifiedRawResponse;
+                  try {
+                    stringifiedRawResponse = JSON.stringify(rawResponse);
+                  } catch (error) {
+                    throw error;
```

Comment:
@jcger What is the use of try/catch if you throw the error again? I thought @XavierM meant to handle the error case more gracefully 🤔

Reply:
You are right.

I think I will rollback and use it without any try/catch as checking for [JSON.stringify exceptions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#exceptions) it says that only will throw an error when using a BigInt and if there is a circular reference. The BigInt is catched sooner in the API and we aren't generating any circular reference so we should be fine

Reply:
How about setting a message for `stringifiedRawResponse` like `error in parsing response` with some context about what went wrong?

Reply:
rawResponse comes directly from the backend so there will be no circular reference nor a BigInt at this point. They would throw an error before showing an error toast. As those are the only way to make JSON.stringify to throw an error, there is no need to do anything more

Reply:
I added server side error handling